openssl/src/dh.rs

```diff
@@ -77,15 +77,18 @@ impl Dh<Params> {
     ///
     /// This corresponds to [`DH_generate_parameters`].
```

Comment:
```suggestion
    /// This corresponds to [`DH_generate_parameters_ex`].
```

Reply:
Good catch! That's an excellent improvement. I've re-pushed the branch with this change included. Thank you!